app/screens/CreateAccountScreen.tsx

```diff
@@ -17,14 +17,17 @@ export const CreateAccountScreen: FC<CreateAccountScreenProps> = observer(
     const formikRef = useRef(null)
 
     // Pull in one of our MST stores
-    const { userStore, channelStore } = useStores()
+    const { userStore } = useStores()
 
     // Pull in navigation via hook
     const navigation = useNavigation()
 
     const signup = (data: { displayName: string; username: string; about: string }) => {
-      channelStore.createDefaultChannels()
-      userStore.signup(data.username, data.displayName, data.about)
+      if (!data.displayName) {
+        alert("Display name is required")
```

Comment:
i think it might be a better UX to *make up* a display name rather than bugging people

- random adjective
- random noun

Reply:
yup, I've made username as default